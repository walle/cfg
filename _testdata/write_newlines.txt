cfg
should
read and escape
newlines
