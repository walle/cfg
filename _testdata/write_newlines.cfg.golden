foo = cfg\nshould\nread and escape\nnewlines
